app/views/spree/taxons/_introduction.html.erb

```diff
@@ -2,7 +2,10 @@
    <div class="row no-gutters">
       <div class="col-xl-8">
          <div class="category-banner-wrap">
-            <%= spree_breadcrumbs(@taxon) %>
+            <ol class="breadcrumb">
```

Comment:
Please call breadcrumbs like this : <%= spree_breadcrumbs(@taxon) %>

Reply:
sir this file is not done in this sprint